fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/client/fluid/FluidVariantRenderHandler.java

```diff
@@ -120,10 +124,12 @@ default int getColor(FluidVariant fluidVariant, @Nullable BlockRenderView view,
 	}
 
 	/**
-	 * Return {@code true} if this fluid should fill tanks from top.
+	 * @deprecated Implement {@link FluidVariantAttributeHandler#isGaseous(FluidVariant)} instead.
+	 * This function will be removed in a future iteration of the API.
 	 */
+	@Deprecated(forRemoval = true)
 	default boolean fillsFromTop(FluidVariant fluidVariant) {
```

Comment:
Fwiw, this is slightly distinct from being gaseous - I could imagine some magical *liquids* in mods that float upwards and fill from the top, so this isn't necessarily bad to have with a fallback to gases.

Reply:
I'll rename `isGaseous` to `flowsUpwards` then, since that is the point. (And this info needs to be available on the server too, hence the move to an attribute).

Reply:
I went for `isLighterThanAir`.